packages/fxa-settings/src/pages/InlineRecoverySetup/index.tsx

```diff
@@ -29,17 +27,13 @@ export type InlineRecoverySetupProps = {
   successfulSetupHandler: () => void;
 };
 
-export const viewName = 'inline-recovery-setup';
-
 const InlineRecoverySetup = ({
   recoveryCodes,
   serviceName,
   cancelSetupHandler,
   verifyTotpHandler,
   successfulSetupHandler,
 }: InlineRecoverySetupProps & RouteComponentProps) => {
-  usePageViewEvent(viewName, REACT_ENTRYPOINT);
```

Comment:
We are preferring Glean events, and @chenba spotted that our legacy metrics do not have mapping for inline routes anyway. Filed https://mozilla-hub.atlassian.net/browse/FXA-9357 to add Glean events for inline routes.